vizro-core/docs/pages/user-guides/layouts.md

```diff
@@ -1,10 +1,15 @@
 # How to change the layout of your page
 
-The [`Page`][vizro.models.Page] model accepts a `layout` argument that enables custom arrangement of charts and components on the screen. This guide shows how to customize the grid specifications in the [`Layout`][vizro.models.Layout].
+The [`Page`][vizro.models.Page] model accepts a `layout` argument that enables custom arrangement of charts and components on the screen. This guide shows how to customize the layout using either the default [`Grid`][vizro.models.Layout] model or the optional [`Flex`][vizro.models.Flex] model inside the `layout` argument of the [`Page`][vizro.models.Page] or the [`Container`][vizro.models.Container].
```

Comment:
```suggestion
The [`Page`][vizro.models.Page] model accepts a `layout` argument that enables custom arrangement of charts and components on the screen. This guide shows how to customize the layout using either the default [`Grid`][vizro.models.Layout] model or the optional [`Flex`][vizro.models.Flex] model inside the `layout` argument of the [`Page`][vizro.models.Page] or the [`Container`][vizro.models.Container].
```

Reply:
Since the second sentence, that starts "This guide..." is quite long, and there's an either/or statement, I think it would make sense to make them two bullets:

```suggestion
This guide shows how to customize the layout with the `layout` argument of the [`Page`][vizro.models.Page] or the [`Container`][vizro.models.Container] for:

* either the default [`Grid`][vizro.models.Layout] model 
* or the optional [`Flex`][vizro.models.Flex] model  
```

Or similar!

Reply:
You could even put anchor links to the parts of the page that cover each so the reader can jump straight to them.